src/images/collections/payment.svg

```diff
@@ -0,0 +1 @@
c.033-2.074 1.725-3.739 3.8-3.739s3.767 1.665 3.8 3.739l.001 7.26v-5.991c0-2.016 1.634-3.651 3.65-3.651s3.651 1.635 3.651 3.651v6.771-4.105c0-1.908 1.546-3.454 3.453-3.455.929 0 1.82.369 2.477 1.026s1.026 1.548 1.026 2.477c.001 6.382-.516 11.427-1.552 15.132-1.249 4.443-2.861 8.776-4.819 12.955" stroke="#212121" stroke-linecap="round" stroke-linejoin="round"/><path d="m129 106.5v50m0-50-6 6m6-6 6 6" stroke="#000" stroke-linecap="round"/><path d="m96 89v18" stroke="#000" stroke-dasharray="2 2"/><path d="m107 100v13" stroke="#000" stroke-dasharray="2 2"/><path d="m150 102v11" stroke="#000" stroke-dasharray="2 2"/><path d="m163 96v14" stroke="#000" stroke-dasharray="2 2"/><circle cx="134" cy="67" fill="#f1af00" r="26.5" stroke="#000"/><circle cx="131" cy="64" fill="#f1af00" r="26.5" stroke="#000"/><circle cx="128" cy="61" fill="#f1af00" r="26.5" stroke="#000"/><circle cx="125" cy="58" fill="#f1af00" r="26.5" stroke="#000"/><circle cx="122" cy="55" fill="#ffee43" r="27"/><g mask="url(#a)"><circle cx="122" cy="55" fill="#ffc836" r="27"/><path clip-rule="evenodd" d="m141.341 36.1603-38.181 38.1805c-5.0337-4.9044-8.16-11.7575-8.16-19.3408 0-14.9117 12.088-27 27-27 7.583 0 14.436 3.1263 19.341 8.1603z" fill="#ffee43" fill-rule="evenodd"/><g stroke="#000"><path d="m112 32v46"/><path d="m102 32v46"/><path d="m122 32v46"/><path d="m132 32v46"/><path d="m142 32v46"/><path d="m98 45h46"/><path d="m98 35h46"/><path d="m98 55h46"/><path d="m98 65h46"/><path d="m98 75h46"/><circle cx="111.886" cy="45.8864" fill="#ffc836" r="1.88636"/><circle cx="131.886" cy="54.8864" fill="#ff8100" r="1.88636"/></g></g><g stroke="#000"><path d="m122.273 76.6364c-12.05 0-21.818-9.7684-21.818-21.8182 0-12.0499 9.768-21.8182 21.818-21.8182" stroke-linecap="round"/><circle cx="122" cy="55" r="26.5"/><path d="m136.5 78.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m130.5 80.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m125.5 82.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m120.5 82.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m115.5 81.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m145.5 69.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m147.5 64.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m148.5 60.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m149.5 56.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m149.5 53.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m148.5 49.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m147.5 45.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m141.5 74.5 11.5 11.5" stroke-dasharray="2 2" stroke-linecap="round"/><path d="m94.5 22v12m-5.5-6h11" stroke-linecap="round"/><path d="m141.5 30v12m-5.5-6h11" stroke-linecap="round"/><path d="m90.5 69v12m-5.5-6h11" stroke-linecap="round"/></g></svg>
```

Comment:
Just checking — have you already run this through svgomg to optimize it?

Reply:
Haven't, will do.